src/api/routes/user.js

```diff
@@ -1,8 +1,23 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
-const UserController = require('../controllers/UserController');
+const checkAuth = require("../middleware/check-auth");
 
-// ... //
+const UserController = require("../controllers/UserController");
 
-module.exports = router;
\ No newline at end of file
+/* User Profile */
+
+router.get(
+  "/me/profile",
+  checkAuth,
+  UserController.getSelfProfile,
+  UserController.getProfile
```

Comment:
Once you make the change to getSelfProfile, make sure to remove the second call to getProfile from here as well.